src/app/shell/workshop-details/workshop-page/workshop-page.component.ts

```diff
@@ -14,13 +15,13 @@ import { imgPath } from 'src/app/shared/models/carousel.model';
 @Component({
   selector: 'app-workshop-page',
   templateUrl: './workshop-page.component.html',
-  styleUrls: ['./workshop-page.component.scss']
+  styleUrls: ['./workshop-page.component.scss'],
 })
-
 export class WorkshopPageComponent implements OnInit, OnDestroy {
-  
   readonly Role: typeof Role = Role;
   public categoryIcons = CategoryIcons;
+  selectedIndex: number;
```

Comment:
can you move it under Input/Output, next to the other public fields? According to the style code, input/output should go before public values

Reply:
updated it